install_config/upgrading/manual_upgrades.adoc

```diff
@@ -602,9 +712,8 @@ files. On a master host, install or update to the latest version of the
 The *openshift-ansible-roles* package provides the latest example JSON files.
 endif::[]
 
-Now update the global *openshift* project by running the following commands as a
-user with *cluster-admin* privileges. It is expected that you will receive
-warnings about items that already exist.
+Now update the global *openshift* project by running the following commands. It
+is expected that you will receive warnings about items that already exist.
```

Comment:
"It is expected..." is expletive. Something like:

"Receiving warning about existing items is expected."

(Even though this was there already.)